lib/domain/dtos/filters/NumericalComparisonDto.js

```diff
@@ -13,7 +13,12 @@
 
 const Joi = require('joi');
 
-exports.NumericalComparisonDto = Joi.object({
-    operator: Joi.string().valid('<', '<=', '=', '>=', '>'),
+const operators = ['<', '<=', '=', '>=', '>'];
+
+exports.NonNegativeIntegerComparisonDto = Joi.object({
+    operator: Joi.string().valid(...operators),
     limit: Joi.number().integer().min(0),
 });
+
+exports.OperatorToFloatLimitComparisonDto = Joi.object(Object
+    .fromEntries(operators.map((operator) => [operator, Joi.number()])));
```

Comment:
Does this belong to this PR?

Reply:
Yes, I need `OperatorToFloatLimitComparisonDto` and previous `NumericComparisonDto` had to generic name, so could be mistaken with new one